src/components/Common/ProfileSelector/SelectButton/SelectButton.tsx

```diff
@@ -37,7 +37,7 @@ export default function SelectButton<T>(props: SelectButtonSpec<T>): JSX.Element
 		props.setValue(getPreviousValue());
 	}
 
-	const arrowStyle = props.direction === 'left' ? 'icon icon-arrow-left w-8 fill-current text-white-white' : 'transform rotate-180 icon icon-arrow-left w-8 fill-current text-white-white';
+	const arrowStyle = props.direction === 'left' ? 'fill-current text-white-white' : 'transform rotate-180 fill-current text-white-white';
```

Comment:
Shouldn't this be a useState with useEffect ? The way I see it, this condition will be reevaluated on every single re-render right now.

Reply:
The a useMemo will do the job

Reply:
Actually yes, a useMemo is perfect!